addons/base_import_module/models/ir_module.py

```diff
@@ -32,8 +34,19 @@ def _import_module(self, module, path, force=False):
         values = self.get_values_from_terp(terp)
 
         unmet_dependencies = set(terp['depends']).difference(installed_mods)
+        is_custom = _is_studio_custom(path)
+        studio_err = "Studio customizations require Studio"
```

Comment:
Don't factor out these expressions, mainly because you do not need to evaluate them if everything is installed.
Anyway, the expression of `is_custom` is never evaluated twice.

Reply:
You're right, did not see that!